packages/modules/src/modules/banners/common/choiceCard/ChoiceCards.tsx

```diff
@@ -28,6 +28,7 @@ interface ChoiceCardProps {
     numArticles?: number;
     content?: BannerTextContent;
     cssCtaOverides?: SerializedStyles;
+    onCtaClick: () => void;
```

Comment:
`onCtaClick` is what fires the CLICK event to ophan. It's already being used by the normal, non-choice cards, CTA